frontend/packages/kubevirt-plugin/integration-tests/tests/utils/templates.ts

```diff
@@ -0,0 +1,254 @@
+export const urlTemplate = {
```

Comment:
IMO, I would rather split these templates into three files and put them to a separate directory